installers/customipxe/errors.go

```diff
@@ -0,0 +1,5 @@
+package customipxe
+
+import "errors"
+
+var ErrEmptyIpxeConfig = errors.New("ipxe config URL or Script must be defined")
```

Comment:
Another nit, one that wasn't caught by gometalinter but is in the spirit of one of its linters: This should be `ErrEmptyIPXEConfig` since its `iPXE` because PXE is an acronym like DNS, ID, IP, ...  and should be all caps. The `i` should also be caps because its a new word. Not a blocker if you just want to ignore though.

Reply:
yeah, that's interesting that the linter didn't catch that. I've updated it regardless.

Reply:
ha!

Reply:
wait what... that was way too fast. I'm pretty sure github sent that to you before I submitted it because it complained about needing a comment for the review as a whole...